src/Model/Filter/Like.php

```diff
@@ -15,7 +15,9 @@ class Like extends Base
         'before' => false,
         'after' => false,
         'mode' => 'or',
-        'comparison' => 'LIKE'
+        'comparison' => 'LIKE',
+        'wildcardAny' => '*',
```

Comment:
True, are we fine with that? As I mentioned before no non-dev user would assume it to be % or _ anyway, they use the default wildcards from all OS and google and alike.